openpype/hosts/maya/api/lib.py

```diff
@@ -2111,6 +2111,34 @@ def reset_frame_range():
     cmds.setAttr("defaultRenderGlobals.startFrame", frame_start)
     cmds.setAttr("defaultRenderGlobals.endFrame", frame_end)
 
+    instances = cmds.ls(
+        "*.id",
+        long=True,
+        type="objectSet",
+        recursive=True,
+        objectsOnly=True
+    )
+    frames_attributes = {
+        'frameStart': frame_start,
+        'frameEnd': frame_end,
+        'handleStart': handle_start,
+        'handleEnd': handle_end
+    }
+
+    for instance in instances:
+        if not cmds.attributeQuery("id", node=instance, exists=True):
+            continue
```

Comment:
This check seems to be redundant since the `cmds.ls` call already filters to only nodes with an `.id` attribute.